designer/client/i18n/translations/en.translation.json

```diff
@@ -398,6 +398,10 @@
     "titleField": {
       "helpText": "Appears above the page title. However, if these titles are the same, the form will only show the page title.",
       "title": "Section title"
+    },
+    "hideTitleField": {
+      "helpText": "If you do not want the section title displayed on any page that's part of this section, tick this box",
```

Comment:
`Tick this box if you do not want the section title to show on any pages in this section`

(minor but matches other content that has been written).